src/cpu-template-helper/src/template/dump/aarch64.rs

```diff
@@ -64,10 +82,16 @@ mod tests {
             KVM_REG_SIZE_U64,
             &0x0000_ffff_0000_ffff_u64.to_le_bytes(),
         ));
+        // CPU templates only supports 32, 64 and 128 bit wide registers, so the following registers
+        // should be excluded from the result.
         v.push(Aarch64RegisterRef::new(KVM_REG_SIZE_U256, &[0x69; 32]));
         v.push(Aarch64RegisterRef::new(KVM_REG_SIZE_U512, &[0x69; 64]));
         v.push(Aarch64RegisterRef::new(KVM_REG_SIZE_U1024, &[0x69; 128]));
         v.push(Aarch64RegisterRef::new(KVM_REG_SIZE_U2048, &[0x69; 256]));
+        // The following registers should be excluded from the result.
+        for &id in REG_EXCLUSION_LIST {
+            v.push(Aarch64RegisterRef::new(id, &vec![0; reg_size(id)]));
+        }
```

Comment:
You can use https://doc.rust-lang.org/std/iter/trait.Extend.html here,
```suggestion
        // The following registers should be excluded from the result.        
        v.extend(REG_EXCLUSION_LIST.iter()
            .map(|id| Aarch64RegisterRef::new(id, &vec![0; reg_size(id)]))
        );
```

Reply:
`Aarch64RegisterVec` doesn't implement `extend()`.

Reply:
Okay this then becomes more of a nit where I'm not opinionated either way.

If you want you could use it by implementing it with:
```
impl Extend<Aarch64RegisterRef<'_>> for Aarch64RegisterVec {
    fn extend<T: IntoIterator<Item=Aarch64RegisterRef<'_>>>(&mut self, iter: T) {
        for elem in iter {
            self.push(elem);
        }
    }
}
```